.github/workflows/cryptid-wasm.yml

```diff
@@ -0,0 +1,55 @@
+name: Bulding cryptid.wasm
+
+on: push
+
+jobs:
+  build:
+    name: build wasm
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: install clang
+      run: sudo bash -c "$(wget -O - https://apt.llvm.org/llvm.sh)"
+    - name: creating symlinks
+      run: |
+        sudo ln -svf /usr/bin/clang-9 /usr/bin/clang
+        sudo ln -svf /usr/bin/wasm-ld-9 /usr/bin/wasm-ld
+        sudo ln -svf /usr/bin/llc-9 /usr/bin/llc
+        sudo ln -svf /usr/bin/llvm-nm-9 /usr/bin/llvm-nm
+        sudo ln -svf /usr/bin/llvm-ar-9 /usr/bin/llvm-ar
+    - name: build wasi-libc
+      run: |
+        git clone https://github.com/WebAssembly/wasi-libc.git
+        cd wasi-libc
+        make WASM_CC=/usr/bin/clang WASM_AR=/usr/bin/llvm-ar WASM_NM=/usr/bin/llvm-nm
+        wget https://github.com/jedisct1/libclang_rt.builtins-wasm32.a
+        sudo mkdir /usr/lib/llvm-9/lib/clang/9.0.0/lib/wasi/
+        sudo mv libclang_rt.builtins-wasm32.a /usr/lib/llvm-9/lib/clang/9.0.0/lib/wasi/
+    - name: building warm
+      run: |
+        git clone https://github.com/cryptid-org/warm.git
+        cd warm/
+        ./make.sh clang "--sysroot=/home/runner/work/cryptid-native/cryptid-native/wasi-libc/sysroot -O2 -flto" llvm-ar
+    - name: building cryptid.wasm
+      run: |
+        git clone https://github.com/cryptid-org/cryptid-native.git
```

Comment:
I overlooked this one in the case of warm, but let me add it now:

The repository the workflow refers to should be cloned/checked out using the standard checkout action:

~~~~YAML
- uses: actions/checkout@v2
~~~~

More information:
  * [Using the checkout action](https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#using-the-checkout-action)
  * Example: [parsegrind/push.yml](https://github.com/cryptid-org/parsegrind/blob/master/.github/workflows/push.yml#L11)

Why use the checkout action instead of `git clone`? The reason is quite simple: this action will always check out the correct commit, regardless from which branch you've pushed to.

Reply:
Thank you for your note!

I made the requested change.